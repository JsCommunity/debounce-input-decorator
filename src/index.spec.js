/* eslint-env jest */

import Adapter from 'enzyme-adapter-react-16'
import forEach from 'lodash/forEach'
import React from 'react'
import { configure, shallow } from 'enzyme'

import { Input, Textarea } from './'

configure({ adapter: new Adapter() })

forEach({ Input, Textarea }, (Component, name) => {
  it(name, () => {
    expect(shallow(React.createElement(Component, {
      debounceTimeout: 250,
      value: '',
    }))).toMatchSnapshot()
  })
})
