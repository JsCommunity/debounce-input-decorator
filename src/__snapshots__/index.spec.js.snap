// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input 1`] = `
<input
  onBlur={[Function]}
  onChange={[Function]}
  onKeyDown={[Function]}
  value=""
/>
`;

exports[`Textarea 1`] = `
<textarea
  onBlur={[Function]}
  onChange={[Function]}
  onKeyDown={[Function]}
  value=""
/>
`;
